### DISCUSSION TITLE
Jak 2 Model List

### DISCUSSION THREAD

**Discussion by dstmnaki** [open-goal/jak-project#3180] [Jak 2]
I came here to give you lads my list of models that i have gone through, with explanation what the model has, this should come in handy for anyone who is looking to mod the game, looking for assets etc

Keep in mind tho, i have not listed every object that a file has, mainly just map models, characters and zoomers

[model list.txt](https://github.com/open-goal/jak-project/files/13301651/model.list.txt)

**Comment by precursorunderground** [comment]
Is there a way to decompile and locate these model files for external use?
